wayland.c

```diff
@@ -400,6 +411,27 @@ bool init_wayland(struct mako_state *state) {
 		}
 	}
 
+	// Set up the cursor. It needs a wl_surface with the cursor loaded into it.
+	// If one of these fail, mako will work fine without the cursor being able to change.
+	state->cursor_theme = wl_cursor_theme_load(NULL, 24, state->shm);
+	if (state->cursor_theme == NULL) {
+		fprintf(stderr, "couldn't find a cursor theme\n");
+		return true;
+	}
+	struct wl_cursor *cursor = wl_cursor_theme_get_cursor(state->cursor_theme, "pointing_hand");
```

Comment:
`pointing_hand` is usually used for links, not for buttons or other clickable UI elements. I'd prefer to use `left_ptr` here.

Reply:
Isn't `left_ptr` the default pointer so this change wouldn't do anything in this case?

Reply:
@fluix-dev Yes, left_ptr is the default cursor. However, if you move from a text box like the Firefox address bar to a notification, currently you won't get left_ptr, and you'll instead get ibeam.

Reply:
Also happens when a notification pops up over a terminal.